app/src/main/java/org/mozilla/fenix/components/FenixSnackbar.kt

```diff
@@ -157,8 +162,20 @@ class FenixSnackbarPresenter(
         actionName: String? = null,
         isError: Boolean = false
     ) {
-        FenixSnackbar.make(view, length, isError).setText(text).let {
+        val snackbar = FenixSnackbar.make(view, length, isError).setText(text).let {
             if (action != null && actionName != null) it.setAction(actionName, action) else it
```

Comment:
Nit: I find this usage of `let` a little confusing, I think it would work better without it.  We're creating a view and then configuring a bunch of properties on it, so `apply` seems like it could work well here.